tests.py

```diff
@@ -863,7 +851,7 @@ def test_cgp_grey_animal_kingdom(self):
         tiger = Candidate('tiger', name='Tiger')
         lynx = Candidate('lynx', name='Lynx')
 
-        expected_winners = set([gorilla, monkey, tiger])
+        expected_winners = {gorilla, monkey, tiger}
```

Comment:
set comprehensions:
https://www.smallsurething.com/list-dict-and-set-comprehensions-by-example/